requirements-test.txt

```diff
@@ -1,2 +1,3 @@
 # For testing Dependency loaders
-django_netjsonconfig
+# TODO: Github link used until openwisp-controller 0.8+ is released.
+-e git://github.com/openwisp/openwisp-controller.git#egg=openwisp_controller
```

Comment:
I just realized this can't be in the 0.3.0 release, need to wait for controller 0.8 release.